suite/tests/client-interface/drxmgr-test.dll.c

```diff
@@ -69,6 +75,10 @@ event_app_instruction(void *drcontext, void *tag, instrlist_t *bb, instr_t *inst
                               IF_NOT_X86_(SPILL_SLOT_MAX + 1) & counterA, 1, 0);
     drx_insert_counter_update(drcontext, bb, inst, SPILL_SLOT_MAX + 1,
                               IF_NOT_X86_(SPILL_SLOT_MAX + 1) & counterB, 3, 0);
+#if defined(AARCH64)
+    drx_insert_counter_update(drcontext, bb, inst, SPILL_SLOT_MAX + 1, SPILL_SLOT_MAX + 1,
+                              &counterC, 3, DRX_COUNTER_64BIT);
```

Comment:
Please add at least one test using DRX_COUNTER_REL_ACQ to exercise that path in some fashion.

Reply:
yep, sorry about that. I should have added a test case for that. 
Added a few additional cases for both 32-bit and 64-bit arm to exercise all the available combinations of flags.